src/Ookii.Dialogs.Wpf/ProgressDialog.cs

```diff
@@ -238,6 +268,20 @@ public bool UseCompactPathsForDescription
             }
         }
 
+        /// <summary>
+        /// Gets the current <see cref="System.Threading.CancellationTokenSource"/> being used by the dialog.
+        /// </summary>
+        [Category("Behavior"), Description("Exposes the current CancellationTokenSource, which can be used in async operations."), DefaultValue(false)]
+        public System.Threading.CancellationTokenSource CancellationSource
```

Comment:
I don't think we want to expose our own `CancellationTokenSource` to the user at all... This would allow the cancellation to be forced from the outside and disconnected from the original `CancellationTokenSource` that owns the `CancellationToken` that we received from the user.

Reply:
I'll change it to private. I was not sure if it would be required in some scenarios